src/main/java/org/ow2/proactive/catalog/repository/CatalogObjectGrantRepository.java

```diff
@@ -41,41 +41,37 @@
     List<CatalogObjectGrantEntity> findCatalogObjectGrantEntitiesByCatalogObjectRevisionEntityIdAndBucketEntityId(
             long catalogObjectId, long bucketId);
 
-    List<CatalogObjectGrantEntity> findCatalogObjectGrantEntitiesByCreatorAndBucketEntityId(String admin,
-            long bucketId);
+    List<CatalogObjectGrantEntity> findCatalogObjectGrantEntitiesByBucketEntityIdAndCatalogObjectRevisionEntityId(
```

Comment:
Same here.

Reply:
Same as the one above.

https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods

https://www.baeldung.com/spring-data-derived-queries